src/input/user_input.rs

```diff
@@ -113,24 +119,34 @@ where
                 {
                     if let Some(length) = length {
                         html! {
-                            <input type="text" class={class} style={style}
-                                value={value}
-                                placeholder={placeholder}
-                                autofocus={autofocus}
-                                autocomplete="off"
-                                oninput={oninput}
-                                maxlength={length.to_string()}
-                            />
+                            <>
+                                <input type="text" class={class} style={style}
+                                    value={value}
+                                    placeholder={placeholder}
+                                    autofocus={autofocus}
+                                    autocomplete="off"
+                                    oninput={oninput}
+                                    maxlength={length.to_string()}
+                                />
+                                <div class="host-network-group-input-input-notice">
+                                    { text!(txt, ctx.props().language, explanation_message) }
+                                </div>
+                            </>
                         }
                     } else {
                         html! {
-                            <input type="text" class={class} style={style}
-                                value={value}
-                                placeholder={placeholder}
-                                autofocus={autofocus}
-                                autocomplete="off"
-                                oninput={oninput}
-                            />
+                            <>
+                                <input type="text" class={class} style={style}
+                                    value={value}
+                                    placeholder={placeholder}
+                                    autofocus={autofocus}
+                                    autocomplete="off"
+                                    oninput={oninput}
+                                />
+                                <div class="host-network-group-input-input-notice">
+                                    { text!(txt, ctx.props().language, explanation_message) }
+                                </div>
```

Comment:
Please change this according to what I said above, plus this would better to be a function the both are commonly using.